src/main/kotlin/kotlinx/kover/KoverPlugin.kt

```diff
@@ -4,83 +4,47 @@
 
 package kotlinx.kover
 
-import groovy.lang.Closure
-import groovy.lang.GroovyObject
-import kotlinx.kover.adapters.collectDirs
+import kotlinx.kover.api.CoverageEngine
```

Comment:
We use star imports in the libraries
![image](https://user-images.githubusercontent.com/2780116/137335247-a886195c-a31c-4c91-88fe-d45726b9960f.png)